packages/synapse-react-client/src/components/SynapsePriceFeatureRow.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Typography } from '@mui/material'
+import React from 'react'
+import { SynapsePriceTableCell } from './SynapsePriceTableCell'
+import { CheckTwoTone } from '@mui/icons-material'
+
+export type SynapsePriceFeatureRowProps = {
+  featureDescription: string
+  standardPlanValue?: boolean | string
```

Comment:
Will render nothing if undefined, a checkmark if true (bug, it's currently if the type is boolean!), or the text if it's a string

Reply:
That's a pretty fun props signature, I like it